PWGCF/Flow/Tasks/FlowTask.cxx

```diff
@@ -633,6 +646,16 @@ struct FlowTask {
       return;
     if (tracks.size() < 1)
       return;
+    registry.fill(HIST("hEventCount"), 1.5);
+    auto bc = collision.bc_as<aod::BCsWithTimestamps>();
+    int currentRunNumber = bc.runNumber();
+    for (auto& ExcludedRun : cfgRunRemoveList.value) {
+      if (currentRunNumber == ExcludedRun) {
+        LOGF(info, "Skipping event from run %d", currentRunNumber);
```

Comment:
This logging will be done on a per collision level
Is it really needed in that way?

Reply:
you are right. let me remove it